tests/pytorch/test_match_chosen_dimensions.py

```diff
@@ -0,0 +1,93 @@
+from __future__ import annotations
+
+import pytest
+import torch
+
+from amltk import (
+    Choice,
+    Component,
+    Fixed,
+    MatchChosenDimensions,
+    Sequential,
+    build_model_from_pipeline,
+)
+from amltk.exceptions import MatchChosenDimensionsError
+from tests.pytorch.common import create_optimizer
+
+
+class TestMatchChosenDimensions:
```

Comment:
Why the class? Nothing against it, just never used it personally

Reply:
I thought it is a bit cleaner. Should I change it into functions?

Reply:
No no, it's fine, just unfamiliar and was wondering if there was a functional reason.